app/_components/ui/CategoryFilter.tsx

```
@@ -91,18 +91,18 @@ export default function CategoryFilter({ activeFilter, onFilterChange, isLoggedI
-              {filter.label}
-            </button>
-          );
+           return (
+             <button
+               key={filter.key}
+               onClick={() => handleFilterClick(filter.key)}
+               className={`relative whitespace-nowrap rounded-full px-4 py-2 text-sm font-medium transition-all active:scale-95 ${isActive
+                 ? 'bg-gray-900 text-white shadow-md'
+                 : 'border border-gray-200 bg-white text-gray-600 hover:bg-gray-50'
+                 }`}
+              >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)